features/web-audio.yml

```diff
@@ -0,0 +1,216 @@
+name: Web Audio
+description: The Web Audio API TODO.
+spec: https://webaudio.github.io/web-audio-api/
+caniuse: audio-api
+status:
+  compute_from: api.AudioContext.AudioContext
```

Comment:
Oh I see, you're using the `compute_from` keyword to get the browser compat data for that key only, while still listing all (most) other compat keys below.

Sorry, I've not been following the `compute_from` work very closely. Can you remind me how we should think about this? To what end do we list `compat_features` if they don't play a role in the final status?

Reply:
The purpose of listing many BCD keys is to provide a (reviewed) Baseline status for individual BCD keys so that that status can be shown on the corresponding MDN page.

The way to think about features now is as more "living" and evolving and not set in stone once created. Small additions to a feature can be added without it affecting the feature's overall status.

This will require more judgment calls about what to include and exclude, as the resulting status is no longer a guardrail.

For this feature, I do wonder if we should split out any of the newer parts, in particular the worklet caught my eye. The nice thing is that we don't need to make final decisions, we can break some things out into separate feedback in response to feedback.

Reply:
Makes sense, thank you.
I would definitely break out the Worklet stuff now. Looking at the Chrome and Firefox implementations, as well as the spec changes over time, worklets didn't exist at first, around 2013, and only really became a thing in 2018.

I know this is ancient history, that both AudioContext and AudioWorklet are baseline hight, and so none of this might matter anyway, but to be more truthful to what happened, I think it would make sense to break a feature called AudioWorklet out of this one.